packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/MeetingMembersWithoutTasks.tsx

```diff
@@ -18,9 +17,9 @@ const headerStyle = {
   paddingTop: 24
 }
 
-const getHeaderText = (meetingType) => {
+const getHeaderText = (meetingType: 'retrospective' | 'action' | 'poker') => {
```

Comment:
shall we use the same way as `TSuggestedActionTypeEnum`, to use the `MeetingTypeEnum` type extracted from `GraphQLEnumType`?

Reply:
yep! the relay-compiler gives us those enums in the generated files so we don't have to reference the server from the client so I'll just pull from there.

```ts
import {
  MeetingMembersWithoutTasks_meeting,
  MeetingTypeEnum
} from 'parabol-client/__generated__/MeetingMembersWithoutTasks_meeting.graphql'
```